src/algorithms/shared/LAGraph.h

```diff
@@ -0,0 +1,1527 @@
+//------------------------------------------------------------------------------
```

Comment:
Let's see exactly which function in here are needed for `cdlp`
incase the list is short we might want to remove this file